worlds/banjo_tooie/__init__.py

```diff
@@ -0,0 +1,1094 @@
+            self.get_location(locationName.JINJOTL4).place_locked_item(item)
+            self.get_location(locationName.JINJOGI2).place_locked_item(item)
+            self.get_location(locationName.JINJOHP1).place_locked_item(item)
+
+            item = self.create_item(itemName.RJINJO)
+            self.get_location(locationName.JINJOMT2).place_locked_item(item)
+            self.get_location(locationName.JINJOMT3).place_locked_item(item)
+            self.get_location(locationName.JINJOMT5).place_locked_item(item)
+            self.get_location(locationName.JINJOJR3).place_locked_item(item)
+            self.get_location(locationName.JINJOJR4).place_locked_item(item)
+            self.get_location(locationName.JINJOWW2).place_locked_item(item)
+
+            item = self.create_item(itemName.BLJINJO)
+            self.get_location(locationName.JINJOGM3).place_locked_item(item)
+            self.get_location(locationName.JINJOTL1).place_locked_item(item)
+            self.get_location(locationName.JINJOHP5).place_locked_item(item)
+            self.get_location(locationName.JINJOCC2).place_locked_item(item)
+            self.get_location(locationName.JINJOIH1).place_locked_item(item)
+            self.get_location(locationName.JINJOIH4).place_locked_item(item)
+            self.get_location(locationName.JINJOIH5).place_locked_item(item)
+
+            item = self.create_item(itemName.PJINJO)
+            self.get_location(locationName.JINJOMT1).place_locked_item(item)
+            self.get_location(locationName.JINJOGM5).place_locked_item(item)
+            self.get_location(locationName.JINJOCC1).place_locked_item(item)
+            self.get_location(locationName.JINJOCC3).place_locked_item(item)
+            self.get_location(locationName.JINJOCC5).place_locked_item(item)
+            self.get_location(locationName.JINJOIH2).place_locked_item(item)
+            self.get_location(locationName.JINJOIH3).place_locked_item(item)
+            self.get_location(locationName.JINJOGI4).place_locked_item(item)
+
+            item = self.create_item(itemName.BKJINJO)
+            self.get_location(locationName.JINJOMT4).place_locked_item(item)
+            self.get_location(locationName.JINJOGM2).place_locked_item(item)
+            self.get_location(locationName.JINJOGM4).place_locked_item(item)
+            self.get_location(locationName.JINJOWW1).place_locked_item(item)
+            self.get_location(locationName.JINJOTL3).place_locked_item(item)
+            self.get_location(locationName.JINJOGI1).place_locked_item(item)
+            self.get_location(locationName.JINJOGI5).place_locked_item(item)
+            self.get_location(locationName.JINJOCC4).place_locked_item(item)
+            self.get_location(locationName.JINJOGI3).place_locked_item(item)
+
+    def allow_jiggies_as_filler(self) -> bool:
+        return self.options.replace_extra_jiggies and self.jiggies_in_pool < self.hard_item_limit
```

Comment:
```suggestion
        return bool(self.options.replace_extra_jiggies) and self.jiggies_in_pool < self.hard_item_limit
```
or
```suggestion
        return bool(self.options.replace_extra_jiggies and self.jiggies_in_pool < self.hard_item_limit)
```
Don't think it should really matter with how they're used, but technically using the options like this in logical expressions means they won't return a `bool` like the function def says.